code/modules/mob/living/carbon/human/human.dm

```diff
@@ -690,6 +690,10 @@
 	else if (href_list["lookitem"])
 		var/obj/item/I = locate(href_list["lookitem"])
 		usr.examination(I)
+	else if (href_list["listitems"])
+		var/obj/item/I = locate(href_list["listitems"])
```

Comment:
So it seems there's no more room left in the regular old `flags` var for another flag.
Should I make it a boolean var on `/obj/item`?